### DISCUSSION TITLE
Feature Request: Configure page timeout

### DISCUSSION THREAD

**Discussion by thielium** [percy/cli#601] [Support]
I recently had an issue with my Percy runs timing out on Github Actions.

I ended up having to patch Percy with a different timeout for Chromium:

Here's the line I touched: https://github.com/percy/cli/blob/ceadd44a520750782f6c72197457b4514370a785/packages/core/src/page.js#L7

If we could parameterize, instead of patch, it'd be mighty helpful.

Thanks!

**Comment by Robdel12** [comment]
Hey @thielium! This isn't configurable by design, if a snapshot is taking longer than 30 seconds to load or resolve network requests there's an issue that should be resolved.

**Comment by thielium** [comment]
Hey @Robdel12 -- I definitely understand that perspective, and my first approach was to try to fix my pages.

However, there are a couple reasons I ended up increasing the timeout:
1. **Hardware constraints**. Some of the environments we run the snapshots on have hardware constraints. They're doing everything correctly, they just can't do it as fast.
2. **Timing didn't add up**. I was having over 30 snapshots timeout after 30 seconds, but when I increased the timeout, the job took less than 10 minutes (that is, not 15 minutes). My best guess is that Chrome was able to cache the assets and keep taking snapshots, but I don't have access to the machine to prove it.

A little more context: 
* We're trying to use the `@percy/storybook` CLI as designed.
* A lot of the dev overhead we're experiencing is going to come from Storybook, and Storybook isn't particularly light. There are minimal gains from optimizing, there.
*  Our users and developers don't have complaints about the performance, and all the network requests are local. 

All we really want is a way to configure Storybook to support the environment it's being executed on.
